code/game/machinery/computer/_computer.dm

```diff
@@ -52,6 +54,29 @@
 /obj/machinery/computer/update_icon()
 	cut_overlays()
 	SSvis_overlays.remove_vis_overlay(src, managed_vis_overlays)
+
+	// this bit of code makes the computer hug the wall its next to
+	var/turf/T = get_turf(src)
+	var/list/move = list(0, 0)
```

Comment:
Make a comment explaining the structure, and consider renaming it to something else that makes it clear it stores the pixel changes